hera_librarian/async_transfers/__init__.py

```diff
@@ -0,0 +1,27 @@
+"""
+Async transfer managers. The only thing that the remote librarians
+really need to know about the stores.
+"""
+
+from .core import CoreAsyncTransferManager
+from .local import LocalAsyncTransferManager
+from .rsync import RsyncAsyncTransferManager
+
+AsyncTransferManagers: dict[int, CoreAsyncTransferManager] = {
+    0: CoreAsyncTransferManager,
+    1: LocalAsyncTransferManager,
+    2: RsyncAsyncTransferManager,
+}
+
+AsyncTransferManagerNames: dict[str, int] = {
+    "core": 0,
+    "local": 1,
+    "rsync": 2,
+}
```

Comment:
The reason that these are separate is because the transfer managers are referenced in the database by those numbers, but in the config file as the strings.

So you go config file (e.g. "core") -> ID = 0, out of the db as 0 -> Object.

It's a little nasty, but it's better than storing a frozen version of the object in the db as that causes awful data migration problems.